.github/workflows/automerge.yaml

```diff
@@ -43,9 +43,47 @@ jobs:
         run: |
           echo "Waiting for ~ 10 mins, PRT to be initiated." && sleep 600
 
+      - name: is PRT check Passed ?
+        run: |
+          counter=0
+          STATUSES=$(curl -s https://api.github.com/repos/${{github.repository}}/statuses/${{ github.head_ref }}  | jq -r 'length')
```

Comment:
Just curious about what's happening here with the length ?

Reply:
There could be instances where the PRT would not get triggered and the status API would be empty, in that case, we wanted to grace stop the execution. That's why I'm checking the length here